x/pairing/keeper/staking.go

```diff
@@ -114,16 +107,6 @@ func (k Keeper) StakeNewEntry(ctx sdk.Context, creator, chainID string, amount s
 			// then we need to change the Copy of StoreCurrentEpochStakeStorage to copy of the pointers only
 			// must also change the unstaking to create a new entry entirely
 
-			// paid the difference to module
-			existingEntry.Stake = amount
-			// we dont change stakeAppliedBlocks and chain once they are set, if they need to change, unstake first
-			existingEntry.Geolocation = geolocation
-			existingEntry.Endpoints = endpointsVerified
-			existingEntry.Moniker = moniker
-			existingEntry.DelegateCommission = delegationCommission
-			existingEntry.DelegateLimit = delegationLimit
-
-			k.epochStorageKeeper.ModifyStakeEntryCurrent(ctx, chainID, existingEntry, indexInStakeStorage)
 			detailsMap := map[string]string{}
 			for _, val := range details {
 				detailsMap[val.Key] = fmt.Sprint(val.Value)
```

Comment:
we need to modify the behaviour when the amount is less than current stake, to align with unbond

Reply:
done